Library/Homebrew/uninstall.rb

```diff
@@ -71,7 +71,7 @@ def self.uninstall_kegs(kegs_by_rack, casks: [], force: false, ignore_dependenci
               excluded_names = Homebrew::API::Formula.all_formulae.keys
               maybe_paths = maybe_paths.reject do |path|
                 # Remove extension only if a file
-                # (f.e. directory with name "openssl@1.1" will be trimmed to "openssl@1")
+                # (e.g. directory with name "openssl@1.1" will be trimmed to "openssl@1")
```

Comment:
I've never heard of `f.e.` until now. Could be a good candidate for a `vale` lint.

Reply:
"exempli gratia" -> "for example" has a few fans, but I'm not one of them
https://english.stackexchange.com/questions/138663/why-e-g-and-not-f-e-why-i-e-and-not-t-i